extensions/components/com_redcore/admin/updates/1.8.4.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @package     Redshopb
+ * @subpackage  Upgrade
+ *
+ * @copyright   Copyright (C) 2012 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Upgrade script for redCORE.
+ *
+ * @package     Redcore
+ * @subpackage  Upgrade
+ * @since       1.5
+ */
+class Com_RedcoreUpdateScript_1_8_4
+{
+	/**
+	 * Performs the upgrade after initial Joomla update for this version
+	 *
+	 * @param   JInstallerAdapter  $parent  Class calling this method
+	 *
+	 * @return  bool
+	 */
+	public function execute($parent)
+	{
+		// Clean old library files because we moved them to the extension folder
+		$files = array(
```

Comment:
not needed, that was taken care of earlier, this is leftover since admin part of the component was still installing languages... :/